### PR TITLE
Tuned baseline for track 3: Muon reaches <3.28 at 3375 steps

### PR DESCRIPTION
**Pull request by nilin**
[single seed, closed for lack of significance]

## Summary

This adds a Track 3 logfile for a tuned Muon baseline run. The only change from the initial baseline is doubling the weight decay based on [this observation](https://github.com/KellerJordan/modded-nanogpt/pull/265#issuecomment-4349298382) by [alint77](https://github.com/alint77). This is not an overall Track 3 SOTA claim.

## Configuration

- Optimizer: pure Muon for block matrix parameters
- Muon lr: `0.025`
- Muon weight decay: `0.025`
- Muon momentum: `0.95`
- Seed: `torch.manual_seed(42)`
- Batch size, dataset, architecture: unchanged from Track 3 baseline
- Train steps: `3500`
- Hardware in submitted run: 4x NVIDIA H100 80GB HBM3
- PyTorch: `2.10.0+cu128`

## Result

Submitted logfile:

- `records/track_3_optimization/results/29e51921-6ae2-4442-81fc-b837ebb93acd.txt`

Validation trajectory near the target:

```text
step:3250/3500 val_loss:3.28861
step:3375/3500 val_loss:3.27816
step:3500/3500 val_loss:3.27244
```

### CODE REVIEW

**Comment by nilin** [comment]
single (hand-picked) seed not sufficient for significance

**Comment by KellerJordan** [comment]
This looks like it may work with more trials
I will run them

**Comment by KellerJordan** [comment]
done https://github.com/KellerJordan/modded-nanogpt/commit/d035e686c3fab8e5f8f72515fe3f4c2932911022#diff-551227b77b683c9ea50df09ee5424d491fa740cbc6bb6aa3f2127142172d1b3fR30

**Comment by nilin** [comment]
reopened by mistake
